upgradeTestRail.sh

```diff
@@ -44,13 +46,20 @@ else
     fi
 fi
 
+
+if [ "$(printf '%s\n' "$version" "$cassandraDeprecationVersion" | sort -V | head -n1)" == "$version" ] && [ "$version" != "$cassandraDeprecationVersion" ]; then
```

Comment:
aren't we checking if cassandra deprecation version is lower than 9?

Reply:
ok, also we need to be sure that it will work for any combinations, like 9.0.0, 9.0, 9, 9.0.1, 9.5, 9.5.0, etc... @shubham-kickdrumtech

Reply:
@gneb It was failing for version 9.0 and 9 which we have fixed and pushed the changes, thanks for pointing this,  please find the attachments:
<img width="774" alt="Screenshot 2024-12-27 at 5 52 51 PM" src="https://github.com/user-attachments/assets/82f58810-9079-4af4-a7ea-78e2669c48fb" />
<img width="854" alt="Screenshot 2024-12-27 at 5 53 15 PM" src="https://github.com/user-attachments/assets/2c25d831-379a-4fd0-b2ae-9230437867be" />
<img width="885" alt="Screenshot 2024-12-27 at 5 53 40 PM" src="https://github.com/user-attachments/assets/f6e0f96f-df50-4337-ad99-01346beddd50" />
<img width="789" alt="Screenshot 2024-12-27 at 5 54 02 PM" src="https://github.com/user-attachments/assets/b64a97be-e61b-4195-b75d-3c8c6cf96148" />
<img width="818" alt="Screenshot 2024-12-27 at 5 54 23 PM" src="https://github.com/user-attachments/assets/485efbdc-9020-4e36-8677-c2bc53da8d4b" />
<img width="786" alt="Screenshot 2024-12-27 at 5 55 40 PM" src="https://github.com/user-attachments/assets/f4fefbbb-0ce7-4cde-9c9e-9dc8f59cd8be" />